lms/static/js/student_account/views/RegisterView.js

```diff
@@ -238,6 +238,31 @@
                             });
                         };
 
+                    $('#register-is_adg_employee-optional-label').html("");
+                    $('#register-company-optional-label').html("");
+                    $('.checkbox-is_adg_employee').insertAfter('.select-city');
+                    $('.select-company').insertAfter('.checkbox-is_adg_employee');
+                    $('.plaintext-honor_code').insertAfter('.register-button');
+                    $('.toggle-form').insertAfter('.plaintext-honor_code');
+                    $('<p>&nbsp;</p>').insertBefore('.toggle-form');
+                    $('.select-company').addClass('hidden');
```

Comment:
Can we use variables for these selectors like     var selectCompany = $('.select-company') if we are using it in multiple functions too.